src/MICore/LaunchOptions.cs

```diff
@@ -1841,8 +1836,8 @@
             if (string.IsNullOrEmpty(this.WorkingDirectory))
                 this.WorkingDirectory = source.WorkingDirectory;
 
-            if (string.IsNullOrEmpty(this.VisualizerFile))
-                this.VisualizerFile = source.VisualizerFile;
+            if (this.VisualizerFiles == null || this.VisualizerFiles.Count == 0)
```

Comment:
>if (this.VisualizerFiles == null || this.VisualizerFiles.Count == 0) [](http://example.com/codeflow?start=12&length=68)

This check doesn't make sense. You want: `if (!string.IsNullOrEmpty(source.VisualizerFile))`